f8a_report/stack_report_main.py

```diff
@@ -33,8 +44,60 @@ def main():
         logger.exception("Error Generating v2 report")
         raise e
 
-    return response
+    return missing_latest_nodes
 
 
 if __name__ == '__main__':
-    main()
+    missing_latest_nodes = main()
+
```

Comment:
Can you please move all this code out of this if condition into another method? This should ideally be used only to define the starting point of the module.